src/components/change-requests/review-change-request/review-change-request.tsx

```diff
@@ -26,13 +29,14 @@ const ReviewChangeRequest: React.FC<ReviewChangeRequestProps> = ({
   const crId = parseInt(id);
   const auth = useAuth();
   const history = useHistory();
-  const [reviewNotes, setReviewNotes] = useState('');
```

Comment:
You mean we don’t have to have `useState` for every single form field?!?!? Lol this is awesome!!